java8/Java8.g4

```diff
@@ -348,7 +348,8 @@ fieldModifier
 	;
 
 variableDeclaratorList
-	:	variableDeclarator (',' variableDeclarator)*
+	:	variableDeclarator
+	|	variableDeclaratorList ',' variableDeclarator
```

Comment:
I thought it comes with better parse performance when turning it into left recursion. Just referring to one of your posts: https://github.com/antlr/grammars-v4/issues/528#issuecomment-304441606